app/src/main/java/com/nowak/wjw/simplecompass/data/location/LocationApiHandler.java

```diff
@@ -15,7 +14,9 @@
 
 import timber.log.Timber;
 
+//todo There’s no clear separation of architecture layer. You made a `domain` package, but inside are Android SDK dependencies.
 public class LocationApiHandler extends LocationCallback {
```

Comment:
I use this class as my data source. Is it OK from the architecture point of view?